Assets/Scripts/RoverCamera.cs

```diff
@@ -38,6 +38,34 @@ public string CameraName
     /// </summary>
     public int StreamHeight { get; set; }
 
+    /// <summary>
+    /// A flattened 3x3 matrix representing the intrinsic parameters of this
+    /// camera.
+    /// </summary>
+    public float[] IntrinsicParameters
+    {
+        set
+        {
+            float f = 1.0f;  // focal length can be arbitrary.
```

Comment:
Maybe `f` should be a `SerializeField` property. Seems kinda weird to have a magic number floating about here.